run.sh

```diff
@@ -32,6 +32,10 @@ export CLOUD_AWS_S3_SECRET_KEY=${CLOUD_AWS_S3_SECRET_KEY:=}
 export CLOUD_AWS_S3_REGION=${CLOUD_AWS_S3_REGION:=eu-west-2}
 export XPACK_SECURITY_ENABLE=${XPACK_SECURITY_ENABLE:=false}
 export XPACK_SECURITY_AUDIT_ENABLE=${XPACK_SECURITY_AUDIT_ENABLE:=false}
+export ${XPACK_AUDIT_INDEX_EVENTS_EXCLUDE}=${XPACK_AUDIT_INDEX_EVENTS_EXCLUDE:=}
+export ${XPACK_AUDIT_INDEX_EVENTS_INCLUDE}=${XPACK_AUDIT_INDEX_EVENTS_INCLUDE:="anonymous_access_denied, authentication_failed, realm_authentication_failed, access_granted, access_denied, tampered_request, connection_granted, connection_denied, run_as_granted, run_as_denied"}
```

Comment:
index events include comes with default set - so this could be potentially skipped / or included only when specified. Add extra options above to the README.